Lesson5/Lesson5/Program.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Lesson5
+{
+    public class Rectangle
+    {
+        private int side1;
+        private int side2;
+
+        public Rectangle(int x, int y)
+        {
+            this.side1 = x;
+            this.side2 = y;
+        }
+        
+        public int AreaCalculator()
+        {
+            return this.side1 * this.side2; 
+        }
+
+        public int PerimeterCalculator()
+        {
+            return (this.side1 + this.side2)*2;
+        }
+    }
+
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            int isInt(string value)
```

Comment:
@MariaOsminkina Shouldn't this method be private method in the Program class? no such strange construction in the Main method